### ISSUE TITLE
rerun a11y scan on 8/19

### ISSUE THREAD

**Issue by gbinal** [18F/pulse#477]

**Comment by gbinal** [comment]
Just to affirm, here's how this should look:  

The scan should: 
1. Take the federal domain list
2. Remove legislative and judicial domains 
3. Remove inactive and redirecting domains 
4. Scan the remaining sites 
5. Remove [warnings, notices, and certain of the error scans](https://github.com/18F/pulse/issues/453#issuecomment-240507606)
6. The results should then be used to calculate the error totals and averages on the [agency page](https://staging.pulse.cio.gov/accessibility/agencies/).  

The average number of errors should be measured by taking the number of errors _after_ step 5 and dividing it by the number of domains for an agency _after_ step 3.

**Comment by gbinal** [comment]
@nickbristow - I'd suggest that we go ahead and begin this today or tomorrow, so that we can have plenty of time to review the results and spot check them thoroughly for the site update.  I'll send you a scheduler to that end if you're game.
